packages/app/src/sandbox/eval/loaders/eval.js

```diff
@@ -41,8 +41,8 @@ export default function(
       `) {` +
       code +
       `\n})`;
-    // eslint-disable-next-line no-eval
-    (0, eval)(newCode).apply(
+    // eslint-disable-next-line no-new-func
+    new Function(`return ${newCode}`)().apply(
```

Comment:
@CompuIves 
I couldn't find details on collaboration and doclets.
Do your prefer complementary inline comments?
E.g. with a link to https://reactjs.org/docs/cross-origin-errors.html#source-maps and/or the issue #667